tests/Fakes/TestError.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+
+namespace WtfPhp\JsonApiErrors\Tests\Fakes;
+
+use Error;
+
+/**
+ * Class TestError
+ * @package WtfPhp\JsonApiErrors\Tests\Fakes
+ */
+class TestError extends Error
```

Comment:
Why do we even need that class? For testing the Exceptions we also just use the Standard PHP Exception class, why not also just use the Error class? 🤔

Reply:
I thought you want to have a custom class for the tests 😄

Reply:
I just wanted to have the two cases (`Exception` and `Error`) in the Tests, but it's fine with the Custom class 👍 Just wanted to ask again. 🙂